cleanup-passes/travis.ts

```diff
@@ -32,56 +31,109 @@ async function cleanupTravisConfig(element: ElementRepo): Promise<void> {
 
   let travis = yaml.safeLoad(travisConfigBlob);
 
-  let needsReview = false;
+  const tools: string[] = [
+    'bower',
+    'polylint',
+    'web-component-tester'
+  ];
+
+  const beforeScript: string[] = [
+    `npm install -g ${tools.join(" ")}`,
+    'bower install',
+    'polylint'
+  ];
 
   // update travis config
   // Add polylint to all elements
   if (Array.isArray(travis.before_script)) {
-    const installStep = 'npm install polylint';
-    const runStep = 'polylint';
+    const bs: string[] = travis.before_script;
+    if (!bs.reduce(
```

Comment:
This would easier to read and no longer with the result of the reduce call extracted out into its own const. You could get rid of the bs const as well.

i.e.:

```javascript
const beforeScriptsEqual = travis.before_script.reduce(
    (acc, s, idx) => { return acc && (beforeScript[idx] === s) }, true)
if (!beforeScriptsEqual) {
```